lib/monitor.js

```diff
@@ -103,7 +118,7 @@ Monitor.prototype.stopSocketsMonitor = function () {
  * module.
  * @class
  */
-function Timer(callback, start) {
+function Timer (callback, start) {
```

Comment:
I saw that you have whitespaces between function names and args in all other places so I added them for consistency. We have whitespaces in our internal projects. I can revert those if you don't like them in constructors.